src/ui/Search/frmsearchreplace.cpp

```diff
@@ -187,6 +187,32 @@ int frmSearchReplace::selectAll(QString string, SearchHelpers::SearchMode search
     return count.toInt();
 }
 
+void frmSearchReplace::setSearchMode(SearchHelpers::SearchMode searchMode)
```

Comment:
These two functions are left-overs from the other command-line functions, aren't they? Since they're not used, please remove them again.

Reply:
Actually I left these methods intentionally. I assumed that such methods might come in handy in the future.